testing/opae-c/test_enum_c.cpp

```diff
@@ -670,3 +620,65 @@ TEST_P(enum_c_mock_p, clone_token02) {
 
 INSTANTIATE_TEST_CASE_P(enum_c, enum_c_mock_p,
                         ::testing::ValuesIn(test_platform::mock_platforms()));
+
+class enum_c_err_p : public enum_c_p {};
+
+
+TEST_P(enum_c_err_p, num_errors) {
+	auto device = platform_.devices[0];
+
+	// fme num_errors
+	ASSERT_EQ(fpgaPropertiesSetNumErrors(filter_, device.fme_num_errors),
+		FPGA_OK);
+	EXPECT_EQ(
+		fpgaEnumerate(&filter_, 1, tokens_.data(), tokens_.size(), &num_matches_),
+		FPGA_OK);
+	EXPECT_EQ(num_matches_, GetNumFpgas());
+
+	DestroyTokens();
+
+	// afu num_errors
+	ASSERT_EQ(fpgaPropertiesSetNumErrors(filter_, device.port_num_errors),
+		FPGA_OK);
+	EXPECT_EQ(
+		fpgaEnumerate(&filter_, 1, tokens_.data(), tokens_.size(), &num_matches_),
+		FPGA_OK);
+	EXPECT_EQ(num_matches_, GetNumFpgas());
+
+	DestroyTokens();
+
+	// invalid
+	ASSERT_EQ(
+		fpgaPropertiesSetNumErrors(filter_, invalid_device_.port_num_errors),
+		FPGA_OK);
+	EXPECT_EQ(
+		fpgaEnumerate(&filter_, 1, tokens_.data(), tokens_.size(), &num_matches_),
+		FPGA_OK);
+	EXPECT_EQ(num_matches_, 0);
+}
+
+INSTANTIATE_TEST_CASE_P(enum_c, enum_c_err_p,
+	::testing::ValuesIn(test_platform::platforms({ "skx-p","dcp-rc" })));
+
+class enum_c_socket_p : public enum_c_p {};
+
+TEST_P(enum_c_socket_p, socket_id) {
```

Comment:
This test class is inheriting the exact same class and is targeting the same platforms as the enum_c_err_p test class. Should probably just have both tests use the same class and name them something generic like enum_c_skx_dcp_p or something.

Reply:
test case should be generic , not having any name like skx, dcp etc